index.html

```diff
@@ -694,17 +694,22 @@ <h2>Terminology</h2>
 <section class="normative">
   <h1>Namespaces</h1>
 
+  <p class="note">
+    The following introduced namespaces are working namespaces for the time the specification has the Draft status. 
```

Comment:
Better call them out as "placeholders" and talk about "final namespaces" that will be used after CR.

We could call them "2020-tentative/wot/td/v11"

Reply:
good idea. Just updated the note